app/src/main/java/com/codesoom/assignment/TaskManager.java

```diff
@@ -0,0 +1,62 @@
+package com.codesoom.assignment;
+
+
+import com.codesoom.assignment.Exception.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Task 목록 정보를 저장하고 있으며, Task 목록에 대한 CRUD 처리를 담당합니다.
+ */
+public class TaskManager {
```

Comment:
`TaskManager`가 `Task` 목록 정보를 저장하고 있는지는 주석에 있어 중요한 정보가 아닙니다. 중요한 것은 `TaskManager`가 `Task`를 관리하는 책임을 갖고 있다는 사실이죠. 다음과 같이 주석을 수정하면 `Task`를 저장하는 곳을 DB로 옮긴다 해도 주석을 수정할 일이 없을 것입니다.

```suggestion
/**
 * Task에 대한 CRUD 처리를 담당합니다.
 */
public class TaskManager {
```

Reply:
제가 너무 현재 클래스만 생각하면서 javadoc을 작성했네요.